cobra/core/model.py

```diff
@@ -938,3 +932,52 @@ def __exit__(self, type, value, traceback):
         """Pop the top context manager and trigger the undo functions"""
         context = self._contexts.pop()
         context.reset()
+
+    def merge(self, right, prefix_existing=None, inplace=True,
+              objective='left'):
+        """Merge two models to create a model with the reactions from both
+        models.
+
+        Custom constraints and variables from right models are also copied
+        to left model, however note that, constraints and variables are
+        assumed to be the same if they have the same name.
+
+        right : cobra.Model
+            The model to add reactions from
+        prefix_existing : string
+            Prefix the reaction identifier in the right that already exist
+            in the left model with this string.
+        inplace : bool
```

Comment:
If we are trying to make this context-aware, maybe mention that in the `inplace` argument?